src/routes/blog.js

```diff
@@ -1,9 +1,22 @@
 const express = require("express");
+const path = require("path");
+const fs = require("fs");
 
 const router = express.Router();
 
+const blogPath = path.join(__dirname, "../../data/blog.json");
+
 router.get("/blog", (req, res) => {
```

Comment:
I think, this way will be more beneficial:
https://www.coursera.org/learn/server-side-nodejs/supplement/XIMQE/exercise-instructions-rest-api-with-express-mongodb-and-mongoose-part-1